course/resources/eeb430-syllabus.md

```diff
@@ -0,0 +1,124 @@
+
+# Recommended Textbooks
+Different authors take different approaches to the material. Take the time to read different textbooks and to find out which one best suits your learning style.
+
+## Introductory
+Begon M., C.R. Townsend and J.L. Harper.  2006.  Ecology: from individuals to ecosystems.  4th ed. Blackwell Publishing, Malden, MA, USA. QH541.B43 2006 
+Krebs, C.J.  2009.  Ecology: the experimental analysis of distribution and abundance. 6th ed.  Benjamin Cummings, San Francisco.  QH541.28 K74 2001  
+Ricklefs, R.E. and G.L. Miller.  2000.  Ecology.  4th ed.  New York: W.H. Freeman & Co. QH541.R53 2000X. 
+
+## Advanced
+Vandermeer, J.H. and Goldberg, D.E. Population Ecology. Princeton University Press., Princeton, NJ, USA. QH352 .V36 2003X
+
+# Assignments / Course learning outcomes
+The main goals are to:
+1) integrate and apply concepts and theoretical models learnt in class,
+2) learn how to collect data with real organisms (technical skills), 
+3) learn how to use basic statistics to analyze and interpret data,
+4) realize that data do not always fit theoretical models, but that it is important to interpret data with a larger theoretical framework in mind, 
+5) learn how to work in group. 
+
+The due dates for each lab report are listed on the lab schedule. Hand in a hard copy of your lab reports at the beginning of the lab. 
+
+# Improving your writing skills
+Effective communication is crucial in science. The University of Toronto provides services to help you improve your writing (see specific section on lab reports), from general advices on effective writing to writing centers and writing courses. See http://www.writing.utoronto.ca/. The Faculty of Arts & Science also offers an English Language Learning (ELL) program, which provides free individualized instruction in English skills. Take advantage of these!
+
+# Academic integrity:
+You should be aware of the University of Toronto Code of Behaviour on Academic Matters. Also see http://www.writing.utoronto.ca/advice/using-sources/how-not-to-plagiarize on How Not to Plagiarize. Note that it is NOT appropriate to use large sections from internet sources, and inserting a few words here and there does not make it an original piece of writing. Be careful in using internet sources – there is no review of most online material and there are MANY errors out there. Use only academic or government internet sources when absolutely necessary. Make sure you read material from many sources (published, peer-reviewed, trusted internet sources) and that you write an original text using this information. Always cite your sources. In case of doubt about plagiarism, talk to your instructor.
+
+# Lecture schedule
+| Week | Date   | Topic                                               | Instructor     |
+|------|--------|-----------------------------------------------------|----------------|
+| 1    | Sep 12 | Welcome to the course!                              | Joel           |
```

Comment:
So this means we start on a Thursday... We need to think about what this means for assignments, since the course topics largely are in blocks two related lectures per week (Tue/Thur). I will update with a suggestion.